fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/election/adapters/ElectionSetupViewPagerAdapter.java

```diff
@@ -85,12 +85,18 @@ public void afterTextChanged(Editable s) {
             // On each text change we edit the list of questions
             // and we add or remove the question from the list of filled question
             String questionText = s.toString();
+
+            // Prevents the user from creating two different questions with the same name
+            if (questions.contains(questionText)) {
```

Comment:
Just a question: In line 239, why do we check `!text.isEmpty()` but here we don't?

Reply:
You've spotted a very subtle bug indeed! I didn't insert that check, as the next piece of code adds the string to the `questions` list only when it's not empty (lines 93-98). Also, that method is called when the text is changed, so even if at the starting point it's empty, it's not a problem. However, it could happen that a user writes the first question name, creates a new question, and goes back to the first one, clearing the text.

Unfortunately, even if we add the isEmpty() check as you suggested, there's still another potential issue. A user could write a certain title (e.g. 'A'), create another question, go back to the first one and clear its text. Then the application doesn't allow the user to write the same title ('A') for the second question, as it has already been inserted in the `questions` list and not removed (due to the lines 93-98, which leave the question list unchanged if the text is empty).

So, I need to modify those lines to address this scenario.